### DISCUSSION TITLE
No-go areas turn on / off

### DISCUSSION THREAD

**Discussion by Leeb64** [devemux86/cruiser#672] [Q&A] [answered]
I think it would make sense if you could switch the NoGo Aereas on and off as needed.
E.g.: I am allowed to enter the low emission zone with vehicle 1, but not with vehicle 2...

**Comment by devemux86** [comment, marked as answer]
This is already possible in My collection where can turn on / off no-go areas and they appear / disappear on the map.
Or do you mean something else?

**Comment by devemux86** [comment]
Or do you mean the no-go areas remain on the map with some enabled and some disabled?
However, this way things can get complicated:
- No-go circles are managed in groups, like favorites (as they are points with radius).
So we can change the properties of the group, not each no-go circle.

- No-go polygons are managed individually, like tracks (as they are closed tracks).
So we can change the properties of each area, but in My collection screen.

- Disabled no-go areas should have a different indication, such as color.
But we allow them to select custom colors.

**Reply by Leeb64** [reply]
I discovered it.
However, that is a bit misleading...
"Select on map" means: activate.
I think “activate” would be more logical.

**Reply by devemux86** [reply]
Activation is in "Show on map" menu, meaning that visible participate in routing.

It's the same menu as in all favorites. Essentially we're still handling favorites.

**Reply by Leeb64** [reply]
if you know it, it's good.
However, “activate” or “show+activate” would be much more understandable.
I didn't see it right away...Maybe others have the same problem?
